src/main/java/teammates/common/util/GoogleCloudStorageHelper.java

```diff
@@ -130,12 +128,12 @@ public static String getNewUploadUrl(String callbackUrl) {
 
         while (size > 0) {
             // Make sure not to over-read
-            int bytesToRead = Math.min(size, MAX_READING_LENGTH);
-            InputStream blobStream = new BlobstoreInputStream(blobKey, offset);
-            byte[] array = new byte[bytesToRead];
-
-            blobStream.read(array);
+            byte[] array;
+            try (InputStream blobStream = new BlobstoreInputStream(blobKey, offset)) {
+                array = new byte[Math.min(size, MAX_READING_LENGTH)];
```

Comment:
This is not what I meant. Why not just assign the value of `array` when the variable is initialized?
```java
byte[] array = ...;
try (...) {
    ...
}
```